### PR TITLE
[Merged by Bors] - Persist aggregate accumulator across file batches

### PR DESCRIPTION
**Pull request by nacardin**
Resolves https://github.com/infinyon/fluvio/issues/1623

Currently, the initial accumulator is passed into each file batch rather than using the accumulator from the previous batch. This is not desirable since we want to aggregate the entire specified input.

This fix will persist the accumulator across the entire stream.

### CODE REVIEW

**Comment by sehz** [comment]
Can you more notes what was problem and how this PR fixes the issue?

File: crates/fluvio-spu/src/services/public/stream_fetch_test.rs [hunk 1]
```diff
@@ -1274,17 +1274,9 @@ async fn test_stream_aggregate_fetch_multiple_batch(
...
     ctx.leaders_state().insert(test_id, replica.clone());
 
-    // Aggregate 5 records
-    // These records look like:
-    //
-    // 1
-    // 2
-    // 3
-    // 4
-    // 5
```
Comment (morenol) [inline crates/fluvio-spu/src/services/public/stream_fetch_test.rs:1284]: could you adjust the comments instead of removing them?

**Review by morenol — CHANGES_REQUESTED** [review]
lgtm, just a comment

File: crates/fluvio-spu/src/services/public/stream_fetch_test.rs (hunk 1 above)
Comment (sehz) [reply crates/fluvio-spu/src/services/public/stream_fetch_test.rs:1284]: Yup

**Review by sehz — CHANGES_REQUESTED** [review]
LGTM.  Can you add to changelog?

File: crates/fluvio-spu/src/services/public/stream_fetch_test.rs (hunk 1 above)
Comment (nacardin) [reply crates/fluvio-spu/src/services/public/stream_fetch_test.rs:1284]: updated

**Comment by nacardin** [comment]
> LGTM. Can you add to changelog?

done

**Review by sehz — APPROVED** [review]
LGTM.  Great work!

**Pull request closed without merging**